packages/trader/src/Modules/Trading/Components/Elements/purchase-fieldset.jsx

```diff
@@ -1,13 +1,13 @@
-import classNames     from 'classnames';
-import { Popover }    from 'deriv-components';
-import React          from 'react';
-import PropTypes      from 'prop-types';
+import classNames        from 'classnames';
+import { Popover }       from 'deriv-components';
+import React             from 'react';
+import PropTypes         from 'prop-types';
 // import { localize }   from 'App/i18n';
 // import { PopConfirm } from 'App/Components/Elements/PopConfirm';
-import Fieldset       from 'App/Components/Form/fieldset.jsx';
-import ContractInfo   from 'Modules/Trading/Components/Form/Purchase/contract-info.jsx';
+import Fieldset          from 'App/Components/Form/fieldset.jsx';
+import ContractInfo      from 'Modules/Trading/Components/Form/Purchase/contract-info.jsx';
 // import PurchaseLock   from 'Modules/Trading/Components/Form/Purchase/PurchaseLock';
-import PurchaseButton from 'Modules/Trading/Components/Elements/purchase-button.jsx';
+import PurchaseButton    from 'Modules/Trading/Components/Elements/purchase-button.jsx';
```

Comment:
Nit pick: Seems like unnecessary spacing when lines 5:6 and 9 are commented.

Reply:
Fixed